front/components/vaults/VaultLayout.tsx

```diff
@@ -45,3 +79,85 @@ export function VaultLayout({
     </RootLayout>
   );
 }
+
+function VaultBreadCrumbs({
+  owner,
+  vault,
+  category,
+  dataSource,
+  dataSourceView,
+  parentId,
+}: {
+  owner: WorkspaceType;
+  vault: VaultType;
+  category?: DataSourceViewCategory;
+  dataSource?: DataSourceType;
+  dataSourceView?: DataSourceViewType;
+  parentId?: string;
+}) {
+  const { nodes } = useDataSourceContentNodes({
+    owner,
+    dataSource: dataSource,
```

Comment:
Here `dataSource` can be accessed through `dataSourceView.dataSource`.

Reply:
I thought I could do that, then I looked at the `DataSourceViewType` and there is no such field there... This is why I kept `dataSource` after all. Is there something happening that I don't understand?

Reply:
It was introduced a few hours ago in https://github.com/dust-tt/dust/commit/26baf3fe74848adee61f14e7521cef2a53bb7ef4 and it's on `main` here: https://github.com/dust-tt/dust/blob/8e1d57ecc83940580648c7e0fd5d1de90e767e31/front/lib/resources/data_source_view_resource.ts#L308